examples/annotation_import/image.ipynb

```diff
@@ -20,12 +20,12 @@
       },
```

Comment:
<p>what's the motivation for having this as False instead of None?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/523/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>